package.json

```diff
@@ -55,10 +55,12 @@
         "test:e2e": "nx run-many --target=e2e",
         "test:jest": "NODE_OPTIONS=--experimental-vm-modules jest --config jest.config.json --no-cache",
         "pre-commit": "node build/pre-commit/index.cjs",
-        "lint": "npm run lint:eslint",
+        "lint": "eslint --ignore-path .eslintignore .",
```

Comment:
Do we really need so many lint scripts in the root package.json?
'Lint' and 'lint:fix' it's enough for me.